Source/Core/AudioCommon/OpenALStream.cpp

```diff
@@ -178,13 +178,34 @@ void OpenALStream::SoundLoop()
   surround_capable = false;
 #endif
 
-  u32 ulFrequency = m_mixer->GetSampleRate();
-  numBuffers = SConfig::GetInstance().iLatency + 2;  // OpenAL requires a minimum of two buffers
+  u32 ul_frequency = m_mixer->GetSampleRate();
+  // OpenAL requires a minimum of two buffers
+  num_buffers = SConfig::GetInstance().iNumBuffers;
```

Comment:
As num_buffers can only be changed through the INI file I imagine it won't be a problem. Only those users that really want to change this value will be able to. I do not see a problem with having more choice options, but if needed I believe the best value for num_buffers is three.

Also I believe more buffers (instead of larger buffers) can help SoundTouch as it is possible to change the rate of each buffer more often, but I did not test this.